public/style.css

```diff
@@ -383,11 +383,15 @@ body {
   .sidebar-btn {
     visibility: visible;
   }
-
-  .left {
-    left: -300px;
+  .menu-closed{
+    left:-300px !important;
+  }
+  .open-menu {
+    left: 0px !important;
+  }
+  .left{
+    left:-300px;
   }
```

Comment:
This is still here, remove it.

Reply:
it's still there to keep the menu closed by default in mobile screens